### DISCUSSION TITLE
How to access the Copilot as a student for free?

### DISCUSSION THREAD

**Discussion by AnuraghSarkar** [community/community#20859] [Copilot Conversations]
I have used copilot since day one, but suddenly it became a premium feature. It has been an essence for quick development for me. I have been hearing a lot of answers that I can use copilot for free as a student. Could anybody help me out
![ezgif com-gif-maker](https://user-images.githubusercontent.com/57608055/179724898-a604bd82-b212-48c7-830f-e76f5d31a245.jpg)

**Comment by npmInicola90** [comment]
If you can prove to Github that you are a student, then you can access the copilot for free. You can apply for the student developer pack here: https://education.github.com/discount_requests/student_application. Just fill in the form and wait for a response. If you are not a student, then I am afraid you won’t be able to obtain the package, as you may have well understood, it was exclusively created for students.

**Comment by ezitwelve** [comment]
Sign up for the developer option

**Comment by thaisonzdx** [comment]
You can signing up to [Github Student Developer Pack](https://education.github.com/pack) to get this!
